multiqc/modules/deeptools/bamPEFragmentSize.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+""" MultiQC submodule to parse output from deepTools bamPEFragmentSize """
+
+import logging
+import re
+from collections import OrderedDict
+
+from multiqc import config
+from multiqc.plots import table, linegraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class bamPEFragmentSizeMixin():
+    def parse_bamPEFragmentSize(self):
+        """Find bamPEFragmentSize output. Only the output from --table is supported."""
+        self.deeptools_bamPEFragmentSize = dict()
+        for f in self.find_log_files('deeptools/bamPEFragmentSize'):
+            parsed_data = self.parseBamPEFile(f['f'], f['fn'])
+            for k, v in parsed_data.items():
+                if k in self.deeptools_bamPEFragmentSize:
+                    log.warning("Replacing duplicate sample {}.".format(k))
+                self.deeptools_bamPEFragmentSize[k] = v
+
+            if len(parsed_data) > 0:
+                self.add_data_source(f, section='bamPEFragmentSize')
+        if len(self.deeptools_bamPEFragmentSize) > 0:
+            headersSE = OrderedDict()
+            headersSE["Reads Sampled"] = {'title': 'Reads Sampled'}
+            headersSE["Read Length Minimum"] = {'title': 'Read Len. Min.', 'description': 'Minimum read length'}
+            headersSE["Read Length 1st Quartile"] = {'title': 'Read Len. 1st Qu.', 'description': '1st quartile read length'}
+            headersSE["Read Length Mean"] = {'title': 'Read Len. Mean', 'description': 'Mean read length'}
+            headersSE["Read Length Median"] = {'title': 'Read Len. Median', 'description': 'Median read length'}
+            headersSE["Read Length 3rd Quartile"] = {'title': 'Read Len. 3rd Qu.', 'description': '3rd quartile read length'}
+            headersSE["Read Length Maximum"] = {'title': 'Read Len. Max.', 'description': 'Maximum read length'}
+            headersSE["Read Length Std. Dev."] = {'title': 'Read Len. Std. Dev.', 'description': 'read length standard deviation'}
+            headersSE["Read Length Med. Abs. Dev."] = {'title': 'Read Len. MAD', 'description': 'read length median absolute deviation'}
+            self.add_section(name="bamPEFragmentSize (read length metrics)",
+                             anchor="bamPEFragmentSize",
+                             plot=table.plot(self.deeptools_bamPEFragmentSize, headersSE))
```

Comment:
It's nice to add a third config dict here with `namespace`:
```
config = {
    'namespace': 'deepTools bamPEFragmentSize',
}
```

Then the hover text displays as `deepTools bamPEFragmentSize: Reads Sampled` instead of just `: Reads Sampled` (I should probably update the core function to drop the `:` if no namespace is given).